command/genesis/utils.go

```diff
@@ -163,6 +164,55 @@ func parseBurnContractInfo(burnContractInfoRaw string) (*polybft.BurnContractInf
 	}, nil
 }
 
+type baseFeeInfo struct {
+	baseFee            uint64
+	baseFeeEM          uint64
+	baseFeeChangeDenom uint64
+}
+
+// parseBaseFeeConfig parses provided base fee configuration and returns baseFeeInfo
+func parseBaseFeeConfig(baseFeeConfigRaw string) (*baseFeeInfo, error) {
+	baseFeeInfo := &baseFeeInfo{
+		command.DefaultGenesisBaseFee,
+		command.DefaultGenesisBaseFeeEM,
+		command.DefaultGenesisBaseFeeChangeDenom,
+	}
+
+	baseFeeConfig := strings.Split(baseFeeConfigRaw, ":")
+	if len(baseFeeConfig) > 3 {
+		return baseFeeInfo, errors.New("invalid number of arguments for base fee configuration")
+	}
+
+	if baseFeeConfig[0] != "" {
+		baseFee, err := strconv.ParseUint(baseFeeConfig[0], 10, 64)
+		if err != nil {
+			return baseFeeInfo, err
+		}
+
+		baseFeeInfo.baseFee = baseFee
+	}
+
+	if len(baseFeeConfig) == 2 && baseFeeConfig[1] != "" {
```

Comment:
Beware, in case `baseFeeConfig` consists of the 3 parts (base fee, base fee EM and base fee change denominator), only `baseFeeChangeDenom` is going to be populated (this if would not be satisfied):

```suggestion
	if len(baseFeeConfig) >= 2 && baseFeeConfig[1] != "" {
```